etl-pipeline/processes/grin/download.py

```diff
@@ -79,7 +79,7 @@ def unpack_and_upload_ocr_files(self, file_content):
                 for file in tar_file:
                     self.s3_manager.put_object(
                         object=tar_file.extractfile(file).read(),
-                        key=self.ocr_dir + f"{file.name}",
+                        key=self.ocr_dir + str(file.name),
```

Comment:
I think `file.name` will always be a string here and this could just be `self.ocr_dir + file.name`